public/static/docs/api-reference/open.md

```diff
@@ -0,0 +1,178 @@
+# dvc.api.open()
+
+[Context manager](https://www.python.org/dev/peps/pep-0343/#context-managers-in-the-standard-library)
+to open a tracked file.
+
+## Usage
+
+```py
+import dvc.api
+
+with dvc.api.open(
+        'get-started/data.xml',
+        repo='https://github.com/iterative/dataset-registry'
+        ) as fd:
+    # ... Process data
+```
+
+### Signature
+
+```py
+open(path, repo=None, rev=None, remote=None, mode="r", encoding=None)
+```
+
+### Types
+
+All **parameter** types are
+[string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).
+
+The **return** type is a
+[file object](https://docs.python.org/3/glossary.html#term-file-object).
+
+No exceptions are raised by this function.
+
+## Description
+
+This function is analogous to the
+[`open()`](https://docs.python.org/3/library/functions.html#open) Python
+builtin, but for files tracked in <abbr>DVC projects</abbr> (by DVC or Git).
+However, it may only be used as a _context manager_ (using the `with` keyword as
+shown in the [Examples](#examples) below). There is no `dvc.api.close()`.
+
+> See also `dvc.api.read()` for a shorthand way to read the complete contents of
+> a tracked file.
+
+`dvc.api.open()` avoids downloading files from **most**
+[remote types](/doc/command-reference/remote/add#supported-storage-types). It
+returns an open connection to the storage, though which file can be used
+directly. Only Google Drive storage does not support this, requiring this
+function to completely download the file in `path` (into a temporary directory)
+before the file object is made available.
+
+> This has similar uses as the `dvc get` CLI command.
+
+## Parameters
+
+- **`path`** - specifies the location of the target data within the source
```

Comment:
data -> file

Reply:
location -> path

Reply:
do we need to use term `source` here? do we have a destination?

Reply:
> data -> file

We're using a more generic "target data" here and in --remote to avoid saying "file or directory" for get_url() and just "file" for open()/read(). No strong opinion though, I can change them all differently.

> location -> path

Not sure we want to use the word "path" to define `--path`?

> do we need to use term source

No, removed in a2298039.

Reply:
> data -> file

Alright, using just "file (or directory)" now. 

> location -> path

Changed to "location and file name of ..." now.